bin/ember

```diff
@@ -4,6 +4,9 @@
 // Provide a title to the process in `ps`
 process.title = 'ember';
 
+var autocomplete = require('../lib/utilities/cli-command-completion');
```

Comment:
can this instead be run in cli/index.js ?

changes in `bin/ember` come at a big cost, as `bin/ember` is meant to work across all versions of ember-cli. In essence there is a moratorium on changes to it.